ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -209,8 +208,7 @@ const RenewalCard = withStyle(renewalStyle.card)(
 
 
 // Page to render
-export const AccessRenewalPage = fp.flow(
-  withRouteData,
+export const AccessRenewal = fp.flow(
```

Comment:
Why this change? If this is desired, I think the filename also needs to be updated.

Reply:
Two React components had their `withRouteData` HOC embedded into the component itself instead of the app-routing component. This broke their tests because of how I'm using `withRouteData` to set store values and it's call to `useParams()` was breaking. We can switch this back later once we refactor `withRouteData` but I do think I prefer keeping the routing stuff completely separate from the React component.

The change in name was just to reflect that difference. I can change the filenames as well.